readme.md

```diff
@@ -12,6 +12,7 @@
 - Fetching remote files with optional caching.
 - Publish your workflow to npm.
 - Automatic [update notifications](#update-notifications).
+- Easy [testing](#testing).
```

Comment:
Is this ok? Could be

- Easy to [test workflows](#testing).
- Easily [testable workflows](#testing).